lib/route/link/can.c

```diff
@@ -352,6 +376,10 @@ static int can_put_attrs(struct nl_msg *msg, struct rtnl_link *link)
 
 	nla_nest_end(msg, data);
 
+	if (ci->ci_mask & CAN_HAS_DEVICE_STATS)
+		NLA_PUT(msg, IFLA_INFO_XSTATS, sizeof(ci->ci_device_stats),
+			&ci->ci_device_stats);
```

Comment:
is this useful? There is no public API to set the device-stats, at most you could have gotten it from kernel. But then sending it back, seems wrong.